package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@fermyon/knit-wit",
```

Comment:
nit: should this (and the repo) be `knitwit` instead?

Reply:
I debated between both and hence have it referred to as both (no strong opinions). I am happy to pick one and make it consistent.

Reply:
Once there is a consensus, I will update it to be consistent across the repo.